src/node.js

```diff
@@ -388,6 +396,53 @@
       nextTickQueue.push(obj);
       tickInfo[kLength]++;
     }
+
+    // Internal callbacks that must be run when there is no more micro-tasks
+    // scheduled for this turn.
+    function lateTick(callback) {
+      lateTickQueue.push(callback);
+      scheduleMicrotasks();
```

Comment:
I don't think anything is wrong, if you leave it out you need to wait until the next time the queue is processed which could be setTimeout 100000 as you say.